docs/Glossary/index.md

```diff
@@ -0,0 +1,143 @@
+
+---
+
+## Torrents
+
+### Cross-Seed
+
+The act of [seeding](#seeder) the same torrent files or data across multiple [trackers](#tracker) without re-downloading. It also refers to a [software application of the same name](https://www.cross-seed.org) that automates this process.
+
+### Deluge
+
+A lightweight, free, open-source, cross-platform BitTorrent client.
+
+### qBit Manage
+
+A program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes such as time and tracker.
+
+### qBittorrent
+
+A free, open-source, cross-platform BitTorrent client.
+
+### Leecher
+
+A BitTorrent user who is actively downloading a torrent (referred to as 'leeching'). This term can also describe those who download torrents without sharing them afterward.
+
+### Peer
+
+A general term that encompasses both [Seeders](#seeder) and [Leechers](#leecher).
+
+### Seeder
+
+BitTorrent users who possess the complete set of files in a specific torrent and are sharing them, a process known as 'seeding'.
+
+### Swarm
+
+The collective group of all [Peers](#peer) of a given torrent (this includes both [Seeders](#seeder) and [Leechers](#leecher)) who are actively sharing any portion of the torrent.
+
+### Torrent File
+
+A file containing metadata and integrity information about the files and folders associated with the torrent, along with a list of the network locations for its [trackers](#tracker). Torrent files typically bear the extension `.torrent`.
+
+### Tracker
+
+Servers that assist [Peers](#peer) in locating each other and forming efficient distribution groups known as [swarms](#swarm). The term 'tracker' is also informally used for sites that host [.torrent files](#torrent-file) and operate their own tracker servers.
+
+### Perma-Seed
+
+Seeding your torrents permanently ("forever".) Most often, this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places without taking up any additional hard disk space.
+
+---
+
+## Usenet
+
+### Atomic Moves
+
+This is a real file move, carried out instantaneously, rather than by copying a file from one location to another and then deleting the file from the original folder.
+
+### Instant Moves
+
+Same as [Atomic Moves](#atomic-moves).
+
+### PAR (PAR2) Files
+
+PAR and PAR2 files are "Parity Archive Volume" files primarily used in Usenet as a means to attempt repairing incomplete or corrupted files. The presence of these files, however, is not always a guarantee that "reconstruction" of the original data is possible.
+
+### DMCA
+
+Also known as: "DMCA takedown". The DMCA Takedown process allows creators to send notices to service providers that have material that infringes their copyright.
+When this happens, you often end up with missing/incomplete parts and extraction can subsequently fail.
```

Comment:
this still doesn't make sense to me here 🤔

Reply:
Which part specifically?

Reply:
`When this happens, you often end up with missing/incomplete parts and extraction can subsequently fail.`
why would a DMCA cause you to only get your content partially?

Reply:
They sometimes don't or can't identify the entire release across the many parts, and instead just report a few articles.

Some are more easily identifiable in the case of public posts than private indexers nzbs/uploads, but generally speaking my understanding (I don't really deal with this and don't use public indexers) its not the entire release's articles that are taken down.

Reply:
Ok so this is only really relevant for Usenet, correct?
If that's the case I think it would be a good idea to add that

Reply:
good point